cmake/launchdarklyConfig.cmake

```diff
@@ -0,0 +1,16 @@
+include(CMakeFindDependencyMacro)
+
```

Comment:
Not entirely sure how to support this properly. With find_package, the idea is that the SDK was already built. Now, you may want to decide whether to statically or dynamically link boost/openSSL.

But that depends on how you built the SDK in the first place.

Reply:
Yeah, they basically just need to match.

Unless we start building a bunch of permutations. I don't think that sounds amazing though. (Boost basically does this and it never seems to work when I actually use it.)